### DISCUSSION TITLE
Should one attempt to install a lamp + wordpress to a working ltsp server?

### DISCUSSION THREAD

**Discussion by rkwesk** [ltsp/ltsp#683] [Q&A]
This is not my idea. An it teacher wanted to install a local version of wordpress on a working ltsp server running Ubuntu 18.04.

I had no previous experience of wordpress so I decided to try installing on a clean Ubuntu 20.04 insttall on a standalone pc first.

Because of my inexperience with lamp and wordpress I blindly followed a tutorial I found online which indeed worked in that I could access this from another linux box on the same lan through the browser.

What came to my attention, however, was that I could no longer access the wordpress pc through ssh as I could before. No doubt that the security (php?) changed this. No worries for me but it made me think that such an install on a ltsp server would break something or at least would conflict and prevent wordpress from working.

Can anyone shed light on this?

Richard

**Comment by alkisg** [comment]
Hi Richard,

installing LAMP+Wordpress shouldn't affect the configuration of ssh server. If it did, it would be related to inappropriate instructions in the tutorial that you followed.
So it shouldn't affect the operation of LTSP servers.

That said, at least in schools, we usually want to separate the web server from the LTSP server. It can be on a different machine or e.g. in a VirtualBox VM that runs inside the LTSP server.
Teachers frequently neglect to update their LAMP servers, and they stay in ancient PHP versions and OSes because they don't want to bother finding updates to their plugins etc. If the LAMP server is minimal (no desktop), then at least it won't affect the LTSP server updates...

**Comment by rkwesk** [comment]
Thank you Alki

My thought was also to have another pc as a web server. Thank you for your comments on the ssh issue. No doubt my limited understanding with the tutorial resulted in that. I will try to do it all again and pay more attenttion to whatever security extras there are.

Richard
